readme.md

```diff
@@ -105,23 +113,16 @@ To set up your database and load initial data, run the following commands:
 
 Log into the admin with the credentials ``admin / changeme``.
 
-### Note on demo search:
-
-Because we can't (easily) use ElasticSearch for this demo, we use wagtail's native DB search.
-However, native DB search can't search specific fields in our models on a generalized `Page` query.
-So for demo purposes ONLY, we hard-code the model names we want to search into `search.views`, which is
-not ideal. In production, use ElasticSearch and a simplified search query, per
-[http://docs.wagtail.io/en/v1.8.1/topics/search/searching.html](http://docs.wagtail.io/en/v1.8.1/topics/search/searching.html).
-
-### Heroku deployment:
+Deploy to Heroku
+----------------
 
-If you need to deploy your demo site to a publicly accessible server [Heroku](https://heroku.com)
-provides a one-click deployment solution:
+If you don't want to test locally you can deploy a demo site to a publicly accessible server with [Heroku's](https://heroku.com)
+one-click deployment solution to their free 'Hobby' tier:
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/wagtail/bakerydemo)
 
 If you do not have a Heroku account, clicking the above button will walk you through the steps
-to generate one.  After which, you will be presented with a screen to configure your app. For our purposes,
+to generate one.  At this point you will be presented with a screen to configure your app. For our purposes,
 we will accept all of the defaults and click `Deploy`.  The status of the deployment will dynamically
 update in the browser. Once finished, click `View` to see the public site.
 
```

Comment:
I'm confused by the Heroku recommendation. Why do we choose them rather than any of the other  Django hosts out there? Especially since we go on to warn that your images will probably disappear the next day? How is this acceptable?  And now they have to go set up S3 to solve that problem? Seems like we're recommending a big hassle. I also still don't see how hosting is a concern for our demo site - we're recommending that users NOT start their projects from our codebase - which means they'll only be running our demo locally. Deployment docs all over the web, without favoring a single host. The Django docs have their own links. I know I'm in the minority on this, but it just feels so far away from our layer of concern, favors one host over others, introduces additional dependences (S3)...